draft-irtf-cfrg-hpke.md

```diff
@@ -296,6 +299,89 @@ plaintexts.
 The procedures described in this session are laid out in a
 Python-like pseudocode.  The ciphersuite in use is left implicit.
 
+## Key Schedule
+
+The variants of HPKE defined in this document share a common
+mechanism for translating the inputs to the protocol into an
+encryption context.  The inputs to the key schedule are as follows:
+
+* `pkR` - The receiver's public key
+* `zz` - A shared secret generated via the KEM for this transaction
+* `enc` - An encapsulated key produced by the KEM for the receiver
+* `info` (optional) - Application-supplied information
+* `psk` (optional) - A pre-shared secret held by both the initiator
+  and the receiver.
+* `pskID` (optional) - An identifier for the PSK
+* `pkI` (optional) - The initiator's public key
+
+If any of the optional values is not present (e.g., set to `None`
```

Comment:
```suggestion
If any of the optional values is absent (e.g., set to `None`
```

Reply:
Actually, this whole section needs rewriting after the update.